py-polars/requirements-dev.txt

```diff
@@ -52,6 +52,7 @@ hvplot>=0.9.1
 matplotlib
 # Other
 gevent
+nest_asyncio
```

Comment:
Looks like the project has recently been archived:
https://github.com/erdewit/nest_asyncio

I don't like relying on libraries that are not being maintained. Not sure what the alternative would be, though.

Reply:
This is typically seen as "the" way of handling this issue on Python (I believe even Netflix have a dependency on it), so I'm very curious what's going on there... we should keep an eye on it and try to work out what the rationale is behind the archiving, and maybe vendor if necessary. In the meantime I can also take a look at `unsync`[^1], though that doesn't seem to have received any updates for a while 🤔 

[^1]: https://github.com/alex-sherman/unsync

Reply:
All right, I'll leave this one up to you. If you're comfortable with taking on this dependency, we can go ahead with this 👍

Reply:
Sounds fair to me 🤣